R/skip.R

```diff
@@ -48,7 +48,7 @@
 #'   expect_equal(1, 2)     # this one skipped
 #'   expect_equal(1, 3)     # this one is also skipped
 #' })
-skip <- function(message) {
+skip <- function(message = "Skipping") {
```

Comment:
I don't _love_ providing a default message, I feel like if you call `skip()` then you should justify yourself with a good `message`

I don't feel extremely strong about this though